src/Bicep.Core/Analyzers/Linter/Rules/OutputsShouldNotContainSecretsRule.cs

```diff
@@ -73,10 +75,10 @@ public override void VisitOutputDeclarationSyntax(OutputDeclarationSyntax syntax
         private class OutputValueVisitor : AstVisitor
```

Comment:
There are other rules which check for secrets: ProtectCommandToExecuteSecretsRule, SecretsInParamsMustBeSecureRule, SecureParamsInNestedDeploymentsRule.  Looks like I started to consolidate the code for these in FindPossibleSecretsVisitor, although only one of the rules is currently using it.  How hard would it be to move these changes to FindPossibleSecretsVisitor and change the rule to use it?

If that's difficult, we could add a TODO or a bug for later.  Thx